### ISSUE TITLE
No Parameter definition

### ISSUE THREAD

**Issue by RickyCook** [globality-corp/openapi#10]
I might be reading this wrong, but it seems that there's no `parameter` definition, so it's not possible to fill in a `ParameterList` inline, as it has

```
items': {'oneOf': [{'$ref': '#/definitions/parameter'},
   {'$ref': '#/definitions/jsonReference'}]}
```

```
In [137]: dir(openapi.model)
Out[137]:
[...
 'Oauth2Scopes',
 'Operation',
 'ParameterDefinitions',
 'ParametersList',
 'PathItem',
 'PathParameterSubSchema',
 ...]
```

**Comment by jessemyers** [comment]
You are correct that the implementation does not generate a `Parameter` type. It [omits](https://github.com/globality-corp/openapi/blob/develop/openapi/model.py#L34) all `anyOf` types and just generates the concrete choice values, such as `QueryParameterSubSchema` and `PathParameterSubSchema`.

Your example usage is close; you need [a few more parameters](https://github.com/globality-corp/openapi/blob/develop/openapi/schemas/v2.0/schema.json#L889) to produce a validating query string parameter:

```
model.ParametersList([
    model.QueryParameterSubSchema(**{
    "type": "string",
    "name": "test",
    "in": "query",
    "required": False,
    }),
]).validate()
```

I fully acknowledge that the error message here is lousy; this comes from the `jsonschema` library.

**Comment by RickyCook** [comment]
hmmm yup; that makes sense.

i think since this is such a common use-case, some more examples would really help. i can PR some with query string, and form body (or similar) parameters when i’m done getting my implementation working if you’d like?

**Comment by jessemyers** [comment]
Contributions are definitely appreciated!

I'd start with improving the README?
